tests/infra/integration_server.py

```diff
@@ -19,7 +19,8 @@ def request(self, path, headers=None, data='', method="GET"):
         resp_obj = Response()
         if not headers:
             headers = {}
-        response = request(method, self.domain + path, headers=headers, data=data, allow_redirects=False)
+        url = path if os.environ["API_DOMAIN_NAME"] in path else self.domain + path
```

Comment:
What is the difference between `os.environ["API_DOMAIN_NAME"]` and `self.domain` here?  Could/should they be the same thing?

Also, is it ever possible for `os.environ["API_DOMAIN_NAME"]` to be `None`?

Reply:
> What is the difference between os.environ["API_DOMAIN_NAME"] and self.domain here? Could/should they be the same thing?

In this case domain has the scheme `http://`|`https://` while `os.environ["API_DOMAIN_NAME"]` does not. 

> Also, is it ever possible for os.environ["API_DOMAIN_NAME"] to be None?

Fusillade will not work without know this so no.

Reply:
Looks good to me then.